README.md

```diff
@@ -184,6 +185,53 @@ Note that index templates are not loaded automatically when using outputs other
 
 If data was inserted before this point (eg an opbeans service was started) you'll probably have to delete the auto-created `apm-*` indexes and let them be recreated.
 
+## 🦖 Load testing in Dyno mode
```

Comment:
Wdyt about moving this info to a separate MD document, linking to it from here?

Reply:
I would like to defer this to a larger refactoring of the entire README, but I think it's out of scope for this PR. Would you be OK with me handling it in a follow-up?